ci-operator/jobs/codeready-toolchain/registration-service/OWNERS

```diff
@@ -0,0 +1,22 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- alexeykazakov
+- dipak-pawar
+- matousjobanek
+- michaelkleinhenz
+- nurali-techie
+- sbryzak
+- tinakurian
+- xcoulon
+options: {}
+reviewers:
+- alexeykazakov
+- dipak-pawar
+- matousjobanek
+- michaelkleinhenz
```

Comment:
Can you please remove michaelkleinhenz, as he is not part of openshift organisation. I have already asked him to do the necessary steps for it.